openff/interchange/_tests/_gromacs.py

```diff
@@ -0,0 +1,21 @@
+"""Helpers for testing GROMACS interoperability."""
+
+from openff.utilities import temporary_cd
+
+from openff.interchange import Interchange
+from openff.interchange.components.mdconfig import get_smirnoff_defaults
+from openff.interchange.drivers import get_gromacs_energies
+
+
+def gmx_roundtrip(state: Interchange, apply_smirnoff_defaults: bool = False):
+    with temporary_cd():
+        state.to_gromacs(prefix="state", decimal=8)
```

Comment:
```suggestion
@pytest.mark.parametrize("monolithic", [False, True])
def gmx_roundtrip(state: Interchange, apply_smirnoff_defaults: bool = False, monolithic):
    with temporary_cd():
        state.to_gromacs(prefix="state", decimal=8, monolithic=monolithic)
```

Intended change in #1120 after this is merged in